### PR TITLE
First draft of final exercises for community involvement (chapter 10)

### PR DESCRIPTION
**Pull request by mbonsma**
In brief, these exercises have the learner:
- Review README for jargon 
- Create CODE-OF-CONDUCT, CONTRIBUTING, LICENSE, and NEWS

I referred heavily to Mozilla's [Open Leadership Training material](https://mozilla.github.io/open-leadership-training-series/articles/readme/) which I think is excellent. 

Where applicable I use `usethis` functions, but I'm not sure if I got all the right ones - feedback appreciated!

### CODE REVIEW

File: chapters/10-community-involvement.Rmd [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 - Designing your package to be used by or contributed to from the community
   (refer to Mozilla Science Labs Open Leaders material)
-- Contributing guidelines
 - Being inclusive (code of conduct)
+- Contributing guidelines
```
Comment (mbonsma) [inline chapters/10-community-involvement.Rmd:6]: I switched the order of contributing and code of conduct: not super important, just that code of conduct gets linked to in contributing.

File: chapters/10-community-involvement.Rmd [hunk 2]
```diff
@@ -15,4 +15,37 @@
...
+    - [Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md)
+    - [Django Code of Conduct](https://www.djangoproject.com/conduct/)
+- Include a link to your Code of Conduct in your README.
+ 
+3. Create Contributing Guidelines
+- Create a file called CONTRIBUTING.md by running `usethis::use_tidy_contributing()`. This file will contain information for people interacting with your project about how they can use your package and/or how they can get involved. Here are some example Contributing Guidelines:
+    - [Atom](https://github.com/atom/atom/blob/master/CONTRIBUTING.md)
+    - [Data Carpentry Geospatial R lesson](https://github.com/datacarpentry/r-intro-geospatial/blob/master/CONTRIBUTING.md)
+    - FIXME: more examples
```
Comment (k8hertweck) [inline chapters/10-community-involvement.Rmd:34]: The [tidyverse](https://www.tidyverse.org/contribute/) contributing information is interesting; might be useful to explain that technical info about contributing code is linked at the bottom.

**Review by k8hertweck — APPROVED** [review]
This looks great given the current scope of the chapter! Creating a reprex would be a great exercise too (if we decide to include that here), since it makes people better package managers AND contributors to other packages.

**Review by lwjohnst86 — APPROVED** [review]
Looks good @mbonsma! If its no longer a draft version, you are free to merge it in.

**Pull request merged**